src/app/home/home.page.ts

```diff
@@ -8,42 +8,61 @@ import { Component, OnInit } from '@angular/core';
 export class HomePage implements OnInit {
   constructor() {}
 
-  ngOnInit() {}
+  slides: any[] = [];
+  eventsType: any[] = [];
+
+  ngOnInit() {
+    this.slides = [
```

Comment:
Are the slides only going to have images in them? Without any info? And don’t forget the heart functionality for every slide (to like it), if it’s going to be implemented later, add a comment with TODO for it.
Avoid using common variable names. Isn’t this upcomingEvents ?

Reply:
done